tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -46,45 +44,28 @@ var actionCmd = &cobra.Command{
     Short: "work with actions",
 }
 
-/*
-usage: wsk action update [-h] [-u AUTH] [--docker] [--copy] [--sequence]
-                     [--lib LIB] [--shared [{yes,no}]]
-                     [-a ANNOTATION ANNOTATION] [-p PARAM PARAM]
-                     [-t TIMEOUT] [-m MEMORY]
-                     name [artifact]
-
-positional arguments:
-name                  the name of the action
-artifact              artifact (e.g., file name) containing action
-                    definition
-
-optional arguments:
--h, --help            show this help message and exit
--u AUTH, --auth AUTH  authorization key
---docker              treat artifact as docker image path on dockerhub
---copy                treat artifact as the name of an existing action
---sequence            treat artifact as comma separated sequence of actions
-                    to invoke
---lib LIB             add library to artifact (must be a gzipped tar file)
---shared [{yes,no}]   shared action (default: private)
--a ANNOTATION ANNOTATION, --annotation ANNOTATION ANNOTATION
-                    annotations
--p PARAM PARAM, --param PARAM PARAM
-                    default parameters
--t TIMEOUT, --timeout TIMEOUT
-                    the timeout limit in milliseconds when the action will
-                    be terminated
--m MEMORY, --memory MEMORY
-                    the memory limit in MB of the container that runs the
-                    action
-*/
 var actionCreateCmd = &cobra.Command{
     Use:   "create <name string> <artifact string>",
     Short: "create a new action",
     SilenceUsage:   true,
     SilenceErrors:  true,
     PreRunE: setupClientConfig,
     RunE: func(cmd *cobra.Command, args []string) error {
+
+        if len(args) < 2 {
+            whisk.Debug(whisk.DbgError, "Action create command must have at least two arguments\n")
+            errMsg := fmt.Sprintf("Invalid argument(s). An action name, and artifact are required.")
+            whiskErr := whisk.MakeWskError(errors.New(errMsg), whisk.EXITCODE_ERR_GENERAL,
+                whisk.DISPLAY_MSG, whisk.DISPLAY_USAGE)
+            return whiskErr
+        } else if len(args) > 2 {
+            whisk.Debug(whisk.DbgError, "Action create command must not have more than two arguments\n")
+            errMsg := fmt.Sprintf("Invalid argument(s): %s.", strings.Join(args[2:], ", "))
```

Comment:
I think it would be inaccurate to say that there has been too many arguments passed here. This case only considers how many arguments were passed without the -- prefix. Therefore, we would be telling the user that her, or she has supplied too many arguments without counting how many arguments were passed using the -- prefix.